PyRDF/Node.py

```diff
@@ -61,6 +61,40 @@ def __init__(self, get_head, operation):
         self._cur_attr = "" # Name of the new incoming operation
         self.value = None
 
+    def __getstate__(self):
+        """
+        Converts the state of the current node
+        to a Python dictionary.
+
+        Returns
+        -------
+        Python Dictionary
+            A dictionary that stores all instance variables
+            that represent the current PyRDF node.
+
+        """
+        state_dict = {'children':self.children}
+        if not self.operation:
```

Comment:
why not `if self.operation` and then you remove the return from here?

Reply:
The amount of code is same for both right ? 
```
        state_dict = {'children':self.children}
        if self.operation:
            state_dict['operation_name'] = self.operation.name
            state_dict['operation_args'] = self.operation.args
            state_dict['operation_kwargs'] = self.operation.kwargs
            return state_dict

        return state_dict
```

Reply:
Is this what you meant @diocas ?

Reply:
Yes :)

Reply:
oh sorry, looked too quickly... It's not what I meant.. This is:
```
        state_dict = {'children':self.children}
        if self.operation:
            state_dict['operation_name'] = self.operation.name
            state_dict['operation_args'] = self.operation.args
            state_dict['operation_kwargs'] = self.operation.kwargs

        return state_dict
```

Reply:
Ah ok. Right. Will make this change 👍